apps/src/StudioApp.js

```diff
@@ -193,6 +194,18 @@ var StudioApp = function () {
    */
   this.wireframeShare = false;
 
+  /**
+   * Redux store that might be provided by the app. Initially give it an empty
+   * interface so that we can assume existence.
+   */
+  this.reduxStore_ = {
+    getState: function () {
+      return {};
+    },
+    dispatch: function () {
+    }
+  };
```

Comment:
I like the idea of moving this. I worry that creating a real one gets us away from the intended design that the app owns the store rather than studioApp owning the store. Though maybe that's fine if studioApp is doing reduxy things.